### ISSUE TITLE
[Accessibility]A11y_PowershellEditorServices_Installation_Screenreader:No action performs after click Enter on setting button when NVDA is ON.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#61716]
**Environment Details:**
VSCode Version : 1.25.1
OS Version : Win10
NVDA 2018.2
**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "Extension" button of "ActivityBar"->Enter "PowerShell" in "Search Extension" text field and click on "Install" button.
3.Navigate through keyboard such as "Tab" key.
4.Observed that whether  action performs after click enter on setting button.
**Actual:**
No Action performs after click on setting button ny using Enter/Spacebar/Alt+DownArrow when NVDA is On.This performs by mouse click only.
 
**Expected:**
Action should perform after click enter on setting button as expanded and collapsed when NVDA is On..
**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
**[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)**

**User Impact:**
Keyboard user unable to access the content coming after click on setting button.

**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](url)
Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference:**
[2731265_A11y_PowershellEditorServices_Installation_Screenreader.pptx](https://github.com/Microsoft/vscode/files/2510472/2731265_A11y_PowershellEditorServices_Installation_Screenreader.pptx)

**Comment by isidorn** [comment]
Try with latest vscode insiders, this should be fixed
